internal/healthchecks/builtin_test.go

```diff
@@ -135,7 +148,7 @@ func TestAreObjectsHealthy(t *testing.T) {
 					},
 				},
 			},
-			expectedErr: false,
+			expectedErr: "",
```

Comment:
TLDR: IMO, check error strings instead of "an error occurred" is going to result in unit tests that are way to brittle when depending on a third-party library.

While I get the desire to use a string here, I think this is going to result in much more brittle test cases. There was a thread at some point in the `#olm-dev` channel about this and while I don't have a dog in the fight of "we should vs we shouldn't check error strings in tests", I do think this falls into territory where just checking if an error occurred is more appropriate specifically because the important part of the error message is going to be coming from a function call that we don't control (the third-party library). This means that a simple dependency bump could result in unit test failures if they change the values that are returned by `result.Message`  which could realistically be any string.